data/index.js

```diff
@@ -30,6 +30,18 @@ const connect = (type, cb, data) => {
         });
       });
     }
+    if (type === 'put') {
+      homeData.updateOne(data, { $set: { hostName: 'sarah' } }, (updateErr) => {
+        if (updateErr) throw updateErr;
+        console.log('Data updated');
+      });
+    }
+    if (type === 'delete') {
+      homeData.deleteOne(data, (deleteErr) => {
+        if (deleteErr) throw deleteErr;
+        console.log('Data deleted');
+      });
+    }
```

Comment:
It's good to stay consistent with the legacy codebase. But, when we write our future routes, it's better to not throw errors as Eli had when writing the error first callback patterns. Throwing errors will crash your database, and won't communicate the errors in a meaningful way.